manager/integration/tests/common.py

```diff
@@ -5531,12 +5531,16 @@ def create_backing_image_with_matching_url(client, name, url,
         if url == BACKING_IMAGE_RAW_URL:
             expected_checksum = BACKING_IMAGE_RAW_CHECKSUM
         elif url == BACKING_IMAGE_QCOW2_URL:
-            expected_checksum = BACKING_IMAGE_QCOW2_CHECKSUM
+            if DATA_ENGINE == "v2":
+                expected_checksum = BACKING_IMAGE_RAW_CHECKSUM
+            else:
+                expected_checksum = BACKING_IMAGE_QCOW2_CHECKSUM
```

Comment:
Do we need this check after supporting write_zeros `false` for v2 backing image? @ChanYiLin

Reply:
This is not related to write zero
In v2, even the source is qcow2, we will convert it into raw and then dump into the lvol
so the expected checksum should be raw checksum